src/components/VaccineDetails/VaccineDetails.style.tsx

```diff
@@ -0,0 +1,116 @@
+import styled from 'styled-components';
+import { COLOR_MAP } from 'common/colors';
+import ReactMarkdown from 'react-markdown';
+import { FooterLinkStyles } from 'components/Compare/Compare.style';
+import { LinkButton } from 'components/Button';
+import { materialSMBreakpoint } from 'assets/theme/sizes';
+import { MarkdownContent } from 'components/Markdown';
+
+// export const RecommendationItem = styled.div<{ highlight: boolean }>`
+//   display: flex;
+//   padding: 0.75rem;
+//   align-items: flex-start;
+//   background-color: ${({ highlight }) =>
+//     highlight && `${COLOR_MAP.LIGHT_YELLOW}`};
+//   border-radius: 4px;
+
+//   @media (min-width: ${materialSMBreakpoint}) {
+//     padding: 0.5rem 0.25rem;
+//   }
+// `;
+
+export const HeaderCopy = styled.h2`
+  font-weight: bold;
+  font-size: 1.5rem;
+  margin: 0;
+`;
+
+export const BorderedContainer = styled.div`
```

Comment:
You might want to import the styles instead of copying the styles from recommendations